maintainers/maintainer-list.nix

```diff
@@ -3533,6 +3533,12 @@
     github = "ericnorris";
     githubId = 1906605;
   };
+  "3nt3" = {
```

Comment:
A friend of mine more familiar with this says this should likely be it's own PR.